### ISSUE TITLE
Update to 1.20.2/3

### ISSUE THREAD

**Issue by IrisWasTaken1** [QuiltMC/quilted-fabric-api#129]
The latest version crashes when using with 1.20.2

**Comment by OroArmor** [comment]
See QuiltMC/quilt-standard-libraries#346. This needs to be done first

**Comment by IrisWasTaken1** [comment]
> See [QuiltMC/quilt-standard-libraries#346](https://github.com/QuiltMC/quilt-standard-libraries/pull/346). This needs to be done first

what about this? https://github.com/QuiltMC/quilt-standard-libraries/tree/1.20.2

**Comment by ghost** [comment]
Not sure what that is, seems to be some outdated branch... definitely not 1.20.2-ready.

> This branch is 7 commits behind 1.20.

And last commit in August.

**Comment by OroArmor** [comment]
Yes. The PR I linked is the update.

**Comment by verixyt3** [comment]
![UbHoDgu](https://github.com/QuiltMC/quilted-fabric-api/assets/138714269/f8ab91c3-3a1b-426c-b7dc-f20b5a5e7ddf)

**Comment by tbhtyson** [comment]
I went to modrinth and i saw 1.20.1 and not 1.20.2, but i downloaded it and it doesn't work

**Comment by ghost** [comment]
There is a *pull request* open, which means it's being worked on. 1.20.2 support is *not* released yet, please be patient.

I don't know if it needs anything other than approvals at this point, at a quick look the issues with it seem to have been resolved, but I'm not sure, just wait for the Quilt team.

**Comment by OroArmor** [comment]
QSL for 1.20.2 has released a snapshot. Work on this will begin soon

**Comment by Jaffe2718** [comment]
<img width="1280" alt="2023-11-13 132243" src="https://github.com/QuiltMC/quilted-fabric-api/assets/114588637/6bf56922-3731-48dd-870d-5584479f1cae">

When will a stable version of QFAPI & QSL supporting MC 1.20.2 be released?

**Comment by FirstMegaGame4** [comment]
Please stop asking for the 1.20.2 update. It's in the works since there's a pull request open. Let to QFAPI devs the time they need...

Showing the Modrinth page won't do anything, they know that they didn't released it yet since they didn't finished it yet.

**Comment by CyanChanges** [comment]
> Please stop asking for the 1.20.2 update. It's in the works since there's a pull request open. Let to QFAPI devs the time they need...
> 
> Showing the Modrinth page won't do anything, they know that they didn't released it yet since they didn't finished it yet.

So that what went wrong that make them still not publish 1.20.2 update?
And what can we do while it isn't publish yet?

**Comment by FirstMegaGame4** [comment]
Some changes has been made that create conflicts between Fabric Env and Quilt Env that are hard to resolve, plus, QFAPi maintainers are really busy recently delaying the release of the 1.20.2 version. We can't really do other things at the moment that wait for them to complete de compatibiliy between both envs.

**Comment by FirstMegaGame4** [comment]
> > Please stop asking for the 1.20.2 update. It's in the works since there's a pull request open. Let to QFAPI devs the time they need...
> > 
> > Showing the Modrinth page won't do anything, they know that they didn't released it yet since they didn't finished it yet.
> 
> So that what went wrong that make them still not publish 1.20.2 update?
> And what can we do while it isn't publish yet?

Some changes has been made that create conflicts between Fabric Env and Quilt Env that are hard to resolve, plus, QFAPI maintainers are really busy recently, and all of these things are delaying the release of the 1.20.2 version. We can't really do other things at the moment than waiting for them to finish the compatibiliy between both envs.

**Comment by Santiag09** [comment]
any updates at all? I love what you guys are doing, I just wanna see if i can figure out when i can expect to be able to use it

**Comment by FirstMegaGame4** [comment]
the team is working on it, please wait
it's not because not any commits are pushed that they're not working on it

**Comment by diskria** [comment]
Maybe we need update issue title with "1.20.3"?

**Comment by FirstMegaGame4** [comment]
> Maybe we need update issue title with "1.20.3"?

I think that "Update to 1.20.2/3/4" should fit better now 🙃

**Comment by OroArmor** [comment]
Finished!
